### PR TITLE
Fix uniffi-rs documentation about compilation on chapters prerequisit…

### PR DESCRIPTION
**Pull request by gogo2464**
Fix issue https://github.com/mozilla/uniffi-rs/issues/1484

### CODE REVIEW

File: docs/manual/src/tutorial/Prerequisites.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ In your crate's `Cargo.toml` add:
 
 ```toml
 [dependencies]
-uniffi = { version = "[latest-version]" }
+uniffi = { version = "[latest-version]", features = [ "build" ] }
```
Comment (mhammond) [inline docs/manual/src/tutorial/Prerequisites.md:9]: `[dependencies]` should not use the `build` feature - `[build-dependencies]` should, but already is. What problem are you solving exactly?

Comment (gogo2464) [reply docs/manual/src/tutorial/Prerequisites.md:9]: Uniffi-rs litteraly used to crash until I did so.

Comment (mhammond) [reply docs/manual/src/tutorial/Prerequisites.md:9]: You are going to need to provide more details - this change isn't needed by anyone else, and even if it was, "literally crash" doesn't sound possible.

Comment (gogo2464) [reply docs/manual/src/tutorial/Prerequisites.md:9]: Sorry for the inconvenience. Can I provide more details this week please? I was going to finish the week in order to give more details.

**Comment by gogo2464** [comment]
@mhammond hello,


I did:


```powershell
[package]
name = "cryptatools-core"
version = "0.1.14"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
uniffi = {version = "*", features = [ "cli" ]}
uniffi_macros = "*"
lingua = { version = "1.4.0", default-features = true }
assert_float_eq = "1.1.3"
approx = "*"
bimap = "*"
once_cell = "*"
itertools = "*"
rand = "0.8.5"
uniffi_bindgen = "*"

[build-dependencies]
uniffi = {version = "*", features = [ "cli" ]}

[lib]
name = "cryptatools_core"
crate-type = ["cdylib", "rlib"]
doctest = true
doc = true

[[bin]]
# This can be whatever name makes sense for your project, but the rest of this tutorial assumes uniffi-bindgen.
name = "uniffi-bindgen"
path = "uniffi-bindgen.rs"
```


instead of 


```
[package]
name = "cryptatools-core"
version = "0.1.14"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
uniffi = {version = "*", features = [ "cli" ]}
uniffi_macros = "*"
lingua = { version = "1.4.0", default-features = true }
assert_float_eq = "1.1.3"
approx = "*"
bimap = "*"
once_cell = "*"
itertools = "*"
rand = "0.8.5"
uniffi_bindgen = "*"

[build-dependencies]
uniffi = {version = "*", features = [ "build", "cli" ]}

[lib]
name = "cryptatools_core"
crate-type = ["cdylib", "rlib"]
doctest = true
doc = true

[[bin]]
# This can be whatever name makes sense for your project, but the rest of this tutorial assumes uniffi-bindgen.
name = "uniffi-bindgen"
path = "uniffi-bindgen.rs"
```


and against all my assumption, it has worked this time. no fatal error! It is weird. It failed last time. I am going to run the github action and see once again!

**Comment by heinrich5991** [comment]
Seems to be no problem anymore, so can probably be closed.

**Comment by mhammond** [comment]
I agree, thanks for the prompt!

**Comment by gogo2464** [comment]
I wasted your time. Sorry. Thank for answering!

Yes you can close.

**Pull request closed without merging**